runtime/src/main/cpp/Exceptions.cpp

```diff
@@ -103,6 +103,12 @@ _Unwind_Reason_Code unwindCallback(
 }
 #endif
 
+SourceInfo GetSourceInfo(KConstRef stackTrace, int index) {
```

Comment:
Prefer `lowCamelCase` for private function. `UpperCamelCase` to share C name with kotlin (aka internal interop)